<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScreen</class>
 <widget class="QMainWindow" name="GameScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutMain">
     <item>
      <widget class="QFrame" name="frmMap">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutStatistics">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="layoutRoundStat">
         <item>
          <widget class="QLabel" name="lblCurrentRound">
           <property name="text">
            <string>Current Round:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCurrentRoundNumber">
           <property name="text">
            <string>---------</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCurrentPlayer">
           <property name="text">
            <string>Current Player:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCurrentPlayerName">
           <property name="text">
            <string>---------</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCurrentDice">
           <property name="text">
            <string>Current Dice:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>200</number>
           </property>
           <item>
            <widget class="QLabel" name="lblDice1">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblDice2">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutPlayerStat">
         <item>
          <widget class="QLabel" name="lblPlayer1">
           <property name="text">
            <string>Player1's Acquired Treasures:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblPlayer1Treasures">
           <property name="text">
            <string>-------------</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblPlayer2">
           <property name="text">
            <string>Player2's Acquired Treasures:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblPlayer2Treasures">
           <property name="text">
            <string>-------------</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblPlayer3">
           <property name="text">
            <string>Player3's Acquired Treasures:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblPlayer3Treasures">
           <property name="text">
            <string>-------------</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblPlayer4">
           <property name="text">
            <string>Player4's Acquired Treasures:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblPlayer4Treasures">
           <property name="text">
            <string>-------------</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
